BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -590,22 +591,20 @@ - (void)handleCarouselPushForCategory:(NSString *)categoryName usingPushDetailsD
                     self.deepLinkToCustomPage = [BlueShiftDeepLink deepLinkForRoute:BlueShiftDeepLinkCustomePage];
                     [self.deepLinkToCustomPage performCustomDeepLinking:url];
                     self.blueShiftPushParamDelegate = (id<BlueShiftPushParamDelegate>)[self.deepLinkToCustomPage lastViewController];
-                
-                    // Track notification when the page is deeplinked ...
-                    [self trackAppOpenWithParameters:pushTrackParameterDictionary];
-                
                     if ([self.blueShiftPushParamDelegate respondsToSelector:@selector(handleCarouselPushDictionary: withSelectedIndex:)]) {
                         [self.blueShiftPushParamDelegate handleCarouselPushDictionary:pushDetailsDictionary withSelectedIndex:index];
                     }
                 }
             }
             
             [self setupPushNotificationDeeplink: selectedItem];
+            return;
```

Comment:
will this affect the click tracking call below?

Reply:
On line number 578 we are making click tracking call when we have deeplink url from the selected image. If there no value for selected index or appGourpId mentioned, then the line number 607 click tracking will get called.